lucille-plugins/lucille-tika/pom.xml

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+      <groupId>com.kmwllc</groupId>
+      <artifactId>lucille-plugins</artifactId>
+      <version>0.2.0-SNAPSHOT</version>
+      <relativePath>../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>lucille-tika</artifactId>
+    <packaging>jar</packaging>
+     <name>Lucille Plugins: Lucille Tika</name>
+
+    <properties>
+      <tika.version>2.7.0</tika.version>
+      <maven.compiler.source>11</maven.compiler.source>
+      <maven.compiler.target>11</maven.compiler.target>
+      <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    </properties>
+
+    <dependencies>
+      <dependency>
+        <groupId>com.kmwllc</groupId>
+        <artifactId>lucille-core</artifactId>
+        <version>${project.version}</version>
+        <scope>provided</scope>
+      </dependency>
+
+      <dependency>
+        <groupId>org.slf4j</groupId>
+        <artifactId>slf4j-api</artifactId>
+      </dependency>
+
+      <!-- tika -->
+      <dependency>
+        <groupId>org.apache.tika</groupId>
+              <artifactId>tika-core</artifactId>
+              <version>${tika.version}</version>
```

Comment:
We should think about including the tika project's Bill of Material (BOM) in a dependency management for this module. That would (hopefully) make it less painful for someone downstream to pull this module into an existing codebase.

https://github.com/apache/tika/blob/main/README.md#maven-dependencies

Reply:
Done in new commit